app/lib/katello/concerns/renderer_configuration_extension.rb

```diff
@@ -0,0 +1,21 @@
+module Katello
+  module Concerns
+    module RendererConfigurationExtension
+      extend ActiveSupport::Concern
+
+      module Overrides
+        def allowed_helpers
+          super + [:errata, :host_subscriptions, :host_applicable_errata_ids, :host_applicable_errata_filtered,
+                   :host_latest_applicable_rpm_version, :load_pools, :load_errata_applications, :host_content_facet,
+                   :host_sla, :host_products, :sub_name, :sub_sku, :registered_through, :last_checkin, :host_collections,
+                   :host_subscriptions_names, :host_subscriptions, :host_products_names, :host_collections_names,
+                   :host_redhat_subscription_names, :registered_at]
+        end
+      end
+
+      included do
+        prepend Overrides
+      end
```

Comment:
Not crucial, but can we prepend `::Foreman::Renderer::Configuration` directly instead of including this module which then prepends the `Overrides` module?